core/storage/voiceover/gae_models.py

```diff
@@ -274,8 +265,9 @@ def create_model(
         voice_artist_id: str,
         language_code_to_accent: Dict[str, str]
     ) -> VoiceArtistMetadataModel:
-        """Creates a new VoiceArtistMetadataModel instance. This method will
-        only be used to put only a single instance of the model.
+        """Creates a new VoiceArtistMetadataModel instance.
+        Note that Beam jobs will still be able to modify this model after
+        creation because they bypass this method.
```

Comment:
Suggest adding a new line between the first line of this and "Note that...", since really the Note is a separate point.

Also: creation --> its creation (here and below).

Sorry, I realized while reading this that "creation" as stated is ambiguous, it might be taken as referring to the Beam job.